### ISSUE TITLE
[Enhancement]: Swipe-to-refresh in main tabs

### ISSUE THREAD

**Issue by GloriamSemper** [advplyr/audiobookshelf-app#1256]
### Describe the Feature/Enhancement

I have many podcasts coming in every half hour, often while I have the ABS app open. New arrivals don't immediately show and I have to switch to any other tab and then it refreshes.

A swipe-down-to-refresh feature while in any of the main tabs (Home, Latest, and Library for Podcasts) would be so nice.

### Why would this be helpful?

A swipe-down-to-refresh feature while in any of the main tabs (Home, Latest, and Library for Podcasts) would be so nice to load new podcast episodes without having to switch tabs or exit and re-open the app.

### Future Implementation (Screenshot)

n/a

### Audiobookshelf App Version

Android App - 0.9.74

**Comment by hoggatt** [comment]
Agreed, this would significantly improve the podcast experience. I often have to switch to the main tab, then back to the "latest" tab to refresh the latest results. Pull to refresh (or similar) would greatly improve getting podcasts!
